function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
Commenting here for below:

In your `get_tools_schemas` function, you should cover the case when tool is a list of tools, otherwise it throws an error.

Reply:
In the same function `get_tools_schemas` could you use `issubclass(default, BaseModel)` instead of `isinstance(default, BaseModel)`?

Reply:
The two uses of `isinstance` in that function do not work.

Reply:
fixed